docs/docs/console/kubernetes.md

```diff
@@ -12,13 +12,14 @@ Please follow [these](upgrade-from-v2.1.md) steps before upgrading the managemen
 
 You can install the Management Console on a [single Docker host](docker) or in a dedicated Kubernetes cluster.
 
-## Install the ThreatMapper Management Console
 
-The following instructions explain how to install the ThreatMapper console on a Kubernetes Cluster, and configure external access to the Console.
+# Prerequisites
```

Comment:
```
## Prerequisites
```

This change is required for this
![image](https://github.com/deepfence/ThreatMapper/assets/38394463/6bc73762-a4d3-492b-9005-16f225ac4add)